.github/workflows/python.yml

```diff
@@ -235,8 +248,8 @@ jobs:
               working-directory: ./python
               run: |
                   black --check --diff .
-
     build-amazonlinux-latest:
+        if: (github.repository_owner == 'valkey-io' && github.event_name == 'schedule') || github.event.label.name == 'Core changes'
```

Comment:
Can you talk with @shohamazon - this "github.repository_owner == 'valkey-io'" check doesn't actually check if the PR is opened from other repo. She knows whats the right way to achieve it

Reply:
This is here to make it not run on personal repo. Actually, schedule will always run from main and not on PR, i just don't want that every one that pull this changes into his fork will have schedule as well.